arcgis-ios-sdk-samples/Maps/Display device location with NMEA data sources/README.md

```diff
@@ -6,18 +6,22 @@ Parse NMEA sentences and use the results to show device location on the map.
 
 ## Use case
 
-NMEA sentences can be retrieved from GPS receivers and parsed into a series of coordinates with additional information. Devices without a built-in GPS receiver can retrieve NMEA sentences by using a separate GPS dongle, commonly connected bluetooth or through a serial port.
+NMEA sentences can be retrieved from GNSS/GPS receivers and parsed into a series of coordinates with additional information. iOS devices can retrieve NMEA sentences by using a separate GNSS dongle, commonly connected through Bluetooth or cable.
 
-The NMEA location data source allows for detailed interrogation of the information coming from the GPS receiver. For example, allowing you to report the number of satellites in view.
+The NMEA location data source allows for detailed interrogation of the information coming from the GNSS receiver. For example, allowing you to report the number of satellites in view, accuracy of the location, etc.
 
 ## How to use the sample
 
-Tap "Start" to parse the NMEA sentences into a simulated location data source, and initiate the location display. Tap "Recenter" to recenter the location display. Tap "Reset" to reset the location display.
+Tap "Source" to choose between a simulated location data source or any data source created from a connected GNSS device, and initiate the location display. Tap "Recenter" to recenter the location display. Tap "Reset" to reset the location display and location data source.
 
 ## How it works
 
-1. Load NMEA sentences from a local file.
-2. Parse the NMEA sentence strings, and push data into `AGSNMEALocationDataSource`.
+1. Load NMEA sentences.
+    * If a supported GNSS receiver is connected, the sample can get NMEA updates from it.
+    * Otherwise, the sample will read mock data from a local file.
+2. Create an `AGSNMEALocationDataSource`. There are 2 ways to provide updates to the data source.
+    * When updates are received from a GNSS device or the mock data provider, push the data into `AGSNMEALocationDataSource`.
+    * With the Runtime SDK 100.11.0 release, you can create an `AGSNMEALocationDataSource` with the GNSS device connected via `EAAccessory` and its protocol specified in `UISupportedExternalAccessoryProtocols`. The data source created this way will automatically get updates from the device instead of requiring to push data explicitly.
```

Comment:
```suggestion
    * Starting with Runtime SDK 100.11, you can initialize `AGSNMEALocationDataSource` with a GNSS accessory. The data source created this way will automatically get updates from the device instead of requiring to push data explicitly.
```